packages/sth/package.json

```diff
@@ -14,7 +14,8 @@
     "build:docker": "docker build -t scramjetorg/sth:$(git rev-parse HEAD) -f Dockerfile ../../",
     "clean": "rm -rf ./dist .bic_cache",
     "test": "echo no tests yet -- # npm run test:ava",
-    "test:ava": "ava"
+    "test:ava": "ava",
+    "postinstall": "./src/postinstall.ts"
```

Comment:
Hmmm... Will this change it's location when we build the package? I'd use clean `js` instead.

Reply:
thanks, it has been modified by @patuwwy to:
```
  "postBuildOverride": {
    "scripts": {
      "postinstall": "./postinstall.js"
    }
```
so I mark it as resolved.